airflow/metrics/otel_logger.py

```diff
@@ -55,7 +60,13 @@
 # https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/api.md#updowncounter
 UP_DOWN_COUNTERS = {"airflow.dag_processing.processes"}
 
-METRIC_NAME_PREFIX = "airflow."
+DEFAULT_METRIC_NAME_PREFIX = "airflow"
+# Delimiter is placed between the universal metric prefix and the unique metric name.
+DEFAULT_METRIC_NAME_DELIMITER = "."
+
+
+def full_name(name: str, *, prefix: str = DEFAULT_METRIC_NAME_PREFIX) -> str:
```

Comment:
This used to live in the test module.  I moved it here and use it in both places now.